gcloud-java-storage/src/main/java/com/google/gcloud/storage/Blob.java

```diff
@@ -71,6 +73,21 @@ private BlobSourceOption(StorageRpc.Option rpcOption) {
       }
     }
 
+    private Storage.BlobGetOption toGetOption(BlobInfo blobInfo) {
```

Comment:
Maybe we should rename method above to "toSourceOption" to make method name consistent?

Alternative would be to call this method also `convert` with an extra fields argument.